CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ The types of changes are:
 * Add approval `AuditLog`s for user and sytem approved privacy requests [#1038](https://github.com/ethyca/fidesops/pull/1038)
 * Add finished `AuditLog` for subject requests [#1040](https://github.com/ethyca/fidesops/pull/1040)
 
+* Access Support for Datadog Logs [#1060](https://github.com/ethyca/fidesops/pull/1060)
```

Comment:
Let's move this up so there's no space between this and the other changes. Also put a space after it before the next section.